vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -14345,27 +14835,55 @@ unit:LB-IN2
 unit:LB-MOL
   a qudt:DerivedUnit ;
   a qudt:Unit ;
+  qudt:deprecated true ;
+  dcterms:isReplacedBy unit:MOL_LB ;
   dcterms:description "<p><strong>Pound Mole</strong> is a unit for \\textit{'Mass Amount Of Substance'} expressed as \\(lb-mol\\).</p>."^^qudt:LatexString ;
   qudt:conversionMultiplier 0.45359237 ;
   qudt:expression "\\(lb-mol\\)"^^qudt:LatexString ;
-  qudt:hasDimensionVector qkdv:A1E0L0I0M1H0T0D0 ;
+  qudt:hasDimensionVector qkdv:A1E0L0I0M0H0T0D0 ;
   qudt:hasQuantityKind quantitykind:MassAmountOfSubstance ;
   qudt:iec61360Code "0112/2///62720#UAB402" ;
-  qudt:symbol "lbm⋅mol" ;
-  qudt:ucumCode "[lb_av].mol"^^qudt:UCUMcs ;
-  qudt:uneceCommonCode "P44" ;
+  qudt:symbol "lb-mol" ;
+  qudt:ucumCode "[mol_lb]"^^qudt:UCUMcs ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "Pound Mole"@en ;
+.
+unit:MOL_LB
+  a qudt:DerivedUnit ;
+  a qudt:Unit ;
+  dcterms:description "<p><strong>Pound Mole</strong> is a unit for \\textit{'Mass Amount Of Substance'} expressed as \\(lb-mol\\).</p>."^^qudt:LatexString ;
+  qudt:conversionMultiplier 0.45359237 ;
```

Comment:
...and I believe the conversion multiplier is 1000x bigger, i.e. 453.9...

Reply:
done